flights/ncric_boss3_flight.yaml

```diff
@@ -75,25 +75,7 @@ entityDefinitions:
       ol.id:
         type:  "ol.id" 
         column: "sourceid"
-      ol.name:
-        type: "ol.name"
-        column: "sourcename"
-      ol.description:
-        type: "ol.description"
-        column: "srcdescr"
-      ol.fixedlocation:
-        type: "ol.fixedlocation"
-        column: "fixedmobile"
-        transforms:
-        - !<transforms.ReplaceTransform>
-          target: ["t", "f"]
-          goal: [true, false]
-          ignoreCase: true
-      ol.datasource:
-        type: "ol.datasource"
-        transforms:
-        - !<transforms.ValueTransform>
-          value: "BOSS3"
+    associateOnly: true
```

Comment:
```suggestion
    name: "imagesources"
    associateOnly: true
```

Reply:
The name is still there...  It's below the flag.

Reply:
oh I see it now!